tests/.vagrant/machines/ubuntu/libvirt/action_provision

```diff
@@ -0,0 +1 @@
+1.5:a653a8bb-107e-4a1c-85e0-94e958b85d09
```

Comment:
all those .vagrant files should be removed from the commit.
This makes me think that we should add `.vagrant` to the gitignore